### PR TITLE
enabled running individual browser based tests

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Motivation
https://issues.jboss.org/browse/INTLY-840

## Verification Steps
test the job definition:

```jenkins-jobs --conf jenkins_jobs.ini test jobs/integr8ly/browser-based-testsuite-pipeline.yaml```

go to test pipeline with the same syntax and confirm that single test execution is possible or check the output of this build:

https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/ppaszki-browser-based/13/

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [x] Finished task

### CODE REVIEW

**Comment by trepel** [comment]
@pawelpaszki This is very useful indeed. It just makes me think whether we want to add this to the browser-based-testsuite-pipeline directly or create a separated pipeline (e.g. browser-based-test-executor) for this. If you navigate here: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/browser-based-testsuite-pipeline/ you can see many previous builds on the left-hand side and the test trend graph on the right-hand side. This quickly gives you an idea of what the stability is. If we allow to run just some tests and start using this job for verification etc, we will lose this nice stability view.

ping also @tremes - what do you think, is it worth introducing the separated job ?

**Comment by pawelpaszki** [comment]
@trepel this is very good point actually. I don't mind creating new pipeline specifically for that.

**Comment by tremes** [comment]
+100 for separate pipeline

**Comment by pawelpaszki** [comment]
@trepel @tremes would you mind taking another look? I have moved the job to new .yaml file

File: jobs/integr8ly/browser-based-test-executor.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+
+- job:
+    name: browser-based-code-executor
```
[the remark this answers is not in the document]
Comment (pawelpaszki) [reply jobs/integr8ly/browser-based-test-executor.yaml]: @trepel this is not expected (error). re-running the test myself now

[the remark this answers is not in the document]
Comment (pawelpaszki) [reply jobs/integr8ly/browser-based-test-executor.yaml]: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/browser-based-code-executor/3/console && https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/all/job/browser-based-code-executor/5/console -> both passing one on my branch and one (using your cluster) on master. Not sure why it did not pass the first time

[the remark this answers is not in the document]
Comment (pawelpaszki) [reply jobs/integr8ly/browser-based-test-executor.yaml]: > Mismatch in the name. The filename contains "test" whereas the name here contains "code". I would keed these the same.

changed the pipeline name. Good catch!

**Pull request merged**